modules/RTC/JitsiLocalTrack.js

```diff
@@ -320,6 +335,99 @@ export default class JitsiLocalTrack extends JitsiTrack {
         }
     }
 
+    /**
+     * Starts or stops the effect process and returns the corresponding stream.
+     *
+     * @private
+     * @param {Boolean} enable - Used to start or stop the effect processing
+     * @param {*} effect - Optional parameter represents effect class to create instance
+     * @returns {MediaStream}
+     */
+    _streamEffectEnable(enable: Boolean, effect) {
+        if (enable) {
+            this._effect = effect;
+            this._originalStream = this.stream;
+            this._streamEffect = new this._effect(this._originalStream);
+            this._streamEffect.startEffect();
+
+            return this._streamEffect.getStreamWithEffect();
+        }
+
+        this._streamEffect.stopEffect();
+        this._streamEffect = null;
+
+        return this._originalStream;
+    }
+
+    /**
+     * Switches the video stream associated with this track and updates the containers.
+     *
+     * @private
+     * @param {MediaStream} newStream - The MediaStream to replace
+     * @returns {Promise}
+     */
+    _switchStream(newStream: MediaStream) {
```

Comment:
Since we use `_switchStream` only in `enableEffect`, why don't we include it's code in `enableEffect`?

Reply:
I was thinking that since it is a generic implementation maybe it can be used in the future